channels/ie.m3u

```diff
@@ -1,7 +1,21 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://i.imgur.com/VKolfAl.png" group-title="",Dublin (New!)
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://i.imgur.com/VKolfAl.png" group-title="",Dublin (Webcam)
 http://video3.earthcam.com/fecnetwork/4054.flv/chunklist_w854955906.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/5/5d/Setanta-logo.jpg" group-title="Sport",Setanta Sports
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JE3UAEL.jpg" group-title="Sport",Setanta Sports
 http://ott-cdn.ucom.am/s81/04.m3u8?fluxustv.m3u8
-#EXTINF:-1 tvg-id="TG4.ie" tvg-name="TG4" tvg-logo="https://static.iptv-epg.com/ie/TG4.ie.png" group-title="",TG4
+#EXTINF:-1 tvg-id="TG4.ie" tvg-name="TG4" tvg-logo="https://i.imgur.com/TT9kAEp.png" group-title="",TG4
 http://csm-e.cds1.yospace.com/csm/live/74246610.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/V9cto7E.jpg" group-title="",Oireachtas TV
+https://d1h8rkqumz239n.cloudfront.net/oireachtas/ngrp:oireachtas.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/KNgSr7A.jpg" group-title="",Dáil Éireann
+https://d1h8rkqumz239n.cloudfront.net/dail/ngrp:dail.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/DRPAvjX.jpg" group-title="",Seanad Éireann
+https://d1h8rkqumz239n.cloudfront.net/seanad/ngrp:seanad.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/RNVkCaY.jpg" group-title="",Oireachtas Committee Room 1
+https://d1h8rkqumz239n.cloudfront.net/cr1/ngrp:cr1.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/ZQt5jvM.jpg" group-title="",Oireachtas Committee Room 2
+https://d1h8rkqumz239n.cloudfront.net/cr2/ngrp:cr2.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/5d6WOPx.jpg" group-title="",Oireachtas Committee Room 3
+https://d1h8rkqumz239n.cloudfront.net/cr3/ngrp:cr3.stream_all/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/odMa1KI.jpg" group-title="",Oireachtas Committee Room 4
+https://d1h8rkqumz239n.cloudfront.net/cr4/ngrp:cr4.stream_all/chunklist.m3u8
```

Comment:
Something seems wrong with these streams. They seem to start, but without an image or sound and after 9 seconds they stop working at all.

Reply:
- I tested the links on streamtest.in -> OK
- Tested the streams in Kodi -> OK
- Tested them in VLC player and got the problem you mentioned. , so they seem not to play when you use VLC player.

Reply:
Let’s then agree that we will consider the broadcast to be working only if it is simultaneously launched in both the VLC player and Kodi. As I understand it, these are the most popular free players with support IPTV. Is that all right?

Reply:
It's all right for me